### PR TITLE
deprecated python solo learn and added new content

### PR DESCRIPTION
**Pull request by Lisa1400**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
I can't spot any major issues, just one thing...

- Should the new python solo learn stuff not be plugged into the active syllabuses?

**Review by Ngoakor12 — APPROVED** [review]

File: content/topics/solo-learn/python/intro-to-python/1-getting-started-with-python/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---
+
+Complete the above-mentioned module on Solo Learn.
+
+N.B. When signing up for the course ensure that you sign up for [Introduction to Python](https://www.sololearn.com/learn/courses/python-introduction).
+
+Please submit the link to your SoloLearn profile, for example, https://www.sololearn.com/profile/21320495
+
+We just need to make sure everyone is keeping up 💚
+
```
Comment (ry-oc) [inline content/topics/solo-learn/python/intro-to-python/1-getting-started-with-python/index.md]: Maybe add instructions for the reviewer?

File: content/topics/solo-learn/python/intro-to-python/2-going-deeper-with-python/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/solo-learn/python/intro-to-python/1-getting-started-with-python
```
Comment (ry-oc) [inline content/topics/solo-learn/python/intro-to-python/2-going-deeper-with-python/index.md]: Change this to a soft pre-req or they wont be able to start this until we move the pre-req to complete

File: content/topics/solo-learn/python/python-intermediate/1-collection-types/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+title: Solo Learn - Python Intermediate - 1 Collection Types
+---
+
+Complete the above-mentioned module on Solo Learn.
+
+N.B. When signing up for the course ensure that you sign up for [Python Intermediate](https://www.sololearn.com/learn/courses/python-intermediate).
+
+Please submit the link to your SoloLearn profile, for example, https://www.sololearn.com/profile/21320495
+
+We just need to make sure everyone is keeping up 💚
```
Comment (ry-oc) [inline content/topics/solo-learn/python/python-intermediate/1-collection-types/index.md]: needs instructions for reviewer

File: content/topics/solo-learn/python/python-intermediate/2-functional-programming/index.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+content_type: topic
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/solo-learn/python/python-intermediate/1-collection-types
```
Comment (ry-oc) [inline content/topics/solo-learn/python/python-intermediate/2-functional-programming/index.md]: change to soft pre-req

**Review by ry-oc — COMMENTED** [review]
Good work :) 1 question, are there any points where we confirm that they have completed a course? 

Maybe we should add 2 profile check cards?

**Comment by Lisa1400** [comment]
@ry-oc I didn't change the hard pre-reqs to soft pre-reqs because the hard pre-req links are topics and not projects.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
I think we can be a little bit more specific regarding the instructions for reviewers depending on the profile check card, E.g:
- `Make sure they've submitted a link to their sololearn profile(not just any link)`

For the profile checks cards at the start of courses(first module in intro course, same with first module in intermediate course), include something like:
- `Make sure the course is in the "Courses Progress" section on the sololern profile`

For the end of course profile check cards, include something like:
- `Make sure the course is in the "Certificates" section on the sololern profile`

Otherwise the linter is passing and previous changes seem to have been addressed/explained

**Review by Ngoakor12 — APPROVED** [review]

**Review by ry-oc — CHANGES_REQUESTED** [review]
Looks like you forgot to add the profile check to the syllabuses?

File: content/syllabuses/bootcamp-data-sci-boot-quick-wins.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ _db_id: 61
 title: Data Science boot camp - quick wins
 ---
 
-- {{< contentlink path="topics/solo-learn/python/1-basic-concepts-project" >}}
-- {{< contentlink path="topics/solo-learn/python/2-strings-and-variables" >}}
-- {{< contentlink path="topics/solo-learn/python/3-control-structures-project" >}}
-- {{< contentlink path="topics/solo-learn/python/4-functions-and-modules" >}}
\ No newline at end of file
+- {{< contentlink path="topics/solo-learn/python/intro-to-python/1-getting-started-with-python" >}}
```
Comment (ry-oc) [inline content/syllabuses/bootcamp-data-sci-boot-quick-wins.md:6]: @Lisa1400 I don't see the project on any of the syllabuses

**Review by Ngoakor12 — COMMENTED** [review]
Project checks added to syllabuses :heavy_check_mark: 

Can you also merge develop into this branch to avoid previous linter errors please.

**Comment by Lisa1400** [comment]
@Ngoakor12 I merged develop into my branch. But when I typed `git status` it showed "nothing to commit, working tree clean".

**Comment by Ngoakor12** [comment]
> @Ngoakor12 I merged develop into my branch. But when I typed `git status` it showed "nothing to commit, working tree clean".

You have to make sre you have the latest main code on your machine by pulling first and you also have to push after merging

Try this:

- checkout to `develop` first: `git checkout develop`
- pull main changes locally: `git pull`
- go back to your branch: `git checkout <your-branch>`
- merge develop: `git merge develop`
- push the merged changes: `git push`

**Review by Ngoakor12 — APPROVED** [review]

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**